### DISCUSSION TITLE
Request to create YouTube tutorial videos

### DISCUSSION THREAD

**Discussion by sumithmathew9** [LAPKB/Pmetrics#221] [General]
Dear Michael and Team, I request you to create YouTube tutorial videos (short videos) on how to perform NPAG run, VPC plots, MMopt etc. It will be very helpful for PMetrics users, if you could explain the theory behind some concepts such as MMopt. 
About 10 to 15 min length videos on these topics would be very valuable.

**Comment by mhovd** [comment]
This is something we may consider in the future, but will not happen before the next version of Pmetrics is released. An issue with video tutorials is that they are not "updateable", e.g. if there are changes to the way you use Pmetrics, like those that come in a new version, an entirely new video would have to be made.  For now I have to refer you to the vignettes and articles on the [Pmetrics website](https://lapkb.github.io/Pmetrics/articles/intro.html).

 In addition to the things you mentioned, is there anything else that would be helpful for you to have in a tutorial?
